app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -126,16 +154,63 @@ class ProfileChooserFragmentPresenter @Inject constructor(
   }
 
   private fun bindAddView(binding: ProfileChooserAddViewBinding, @Suppress("UNUSED_PARAMETER") model: ProfileChooserUiModel) {
+    when (whichViewAdd) {
+      true -> {
+        binding.profileChooserAddView.visibility = View.VISIBLE
+        binding.profileChooserAddViewFirstTime.visibility = View.GONE
+      }
+
+      false -> {
+        binding.profileChooserAddViewFirstTime.visibility = View.VISIBLE
+        binding.profileChooserAddView.visibility = View.GONE
+      }
+    }
+
     binding.root.setOnClickListener {
       if (chooserViewModel.adminPin.isEmpty()) {
         activity.startActivity(
-          AdminPinActivity.createAdminPinActivityIntent(activity, chooserViewModel.adminProfileId.internalId, selectUniqueRandomColor())
+          AdminPinActivity.createAdminPinActivityIntent(
+            activity,
+            chooserViewModel.adminProfileId.internalId,
+            selectUniqueRandomColor()
+          )
         )
       } else {
         activity.startActivity(
-          AdminAuthActivity.createAdminAuthActivityIntent(activity, chooserViewModel.adminPin, selectUniqueRandomColor())
+          AdminAuthActivity.createAdminAuthActivityIntent(
+            activity,
+            chooserViewModel.adminPin,
+            selectUniqueRandomColor()
+          )
         )
       }
     }
+
+  }
+
+  private fun getProfileEverCreated(): LiveData<AsyncResult<Boolean>> {
+    return profileManagementController.getWasProfileEverAdded()
+  }
+
+  private fun subscribetoProfileEverCreated(
+    binding: ProfileChooserFragmentBinding
+  ) {
+    profileLiveData.observe(fragment, Observer<AsyncResult<Boolean>> {
```

Comment:
Checkout the implementation of `TopicFragmentPresenter` on how to observe livedata.


Finally what we want it that we have something like this:

`profileLiveData.observe(fragment, Observer<Boolean> {` so that instead of AsyncResult we directly have the value.

Reply:
Done:)